_data/enterprise.yml

```diff
@@ -1,13 +1,13 @@
 System Prerequisites: "/user/enterprise/prerequisites/"
 Installation: "/user/enterprise/installation/"
+Trusty Build Environment: "/user/enterprise/trusty/"
```

Comment:
Why's `trusty-build-containers` becoming `trusty`? I'm a bit afraid that this will lead to confusion between installing on different types of operating systems and our build environments. What about having a structure like this: 

- `<operating system>-build-containers`
- `install-on-<operating system>`

Reply:
@schultyy yeah, I see that concern too. We have `install-on-xenial` for the base-os installer, and "trusty" vs. "precise" fits the rest of the (hosted) docs where os refers to the build environment, so I think it will make it easier to read/connect those are the same things. then, when we have xenial environments and GA base-os xenial , it would be:

*build containers*:`/user/enterprise/xenial/`, `/user/enterprise/trusty/`, and `/user/enterprise/precise/`, to match hosted's`/user/reference/trusty/` and `/user/reference/osx/`
*base os*: `/user/enterprise/install-on-xenial/`, `/user/enterprise/install-on-trusty/`

I think the other option could be:
*build containers*: `/user/enterprise/environments/trusty`, etc
*base os*: `/user/enterprise/trusty`

@plaindocs paging for second opinions?

Reply:
The first option seems more clear to me. And :+1: for consistency.